code/datums/mood_events/generic_positive_events.dm

```diff
@@ -282,6 +282,10 @@
 	mood_change = 2
 	timeout = 3 MINUTES
 
+/datum/mood_event/garland
+	description = "These flowers are rather soothing."
+	mood_change = 1
+
```

Comment:
Next time try not to insert a new mood event in between the same mood event + mood event proc. (you stuck it right between playing_cards and the playing_cards/add_effects)  

Ideally just slap the new mood event at the bottom.